packages/fxa-settings/README.md

```diff
@@ -1,75 +1,182 @@
 
-  // Good!
-  const Button = ({ textColor, children }) => {
-    return <button className={textColor}>{children}</button>;
-  };
-  <Button textColor="text-green-500">Hello, world</Button>;
-  ```
+See the [Tailwind docs on this subject](https://tailwindcss.com/docs/extracting-components/) for more detailed information and examples.
+
+#### PurgeCSS
+
+When it comes to time to create a production build we use PostCSS and PurgeCSS to strip out unused styles. PurgeCSS will look through all of the TSX files and identify which class names to keep styles for. In order for this to work properly it's important to avoid dynamic class names:
+
+```tsx
+// Bad
+const Button = ({ textColor, children }) => {
+  return <button className={`text-${textColor}`}>{children}</button>;
+};
+<Button textColor="green-500">Hello, world</Button>;
+
+// Good!
+const Button = ({ textColor, children }) => {
+  return <button className={textColor}>{children}</button>;
+};
+<Button textColor="text-green-500">Hello, world</Button>;
+```
 
-- If you find a class name is getting stripped out erroneously in production builds, it can be [whitelisted](https://purgecss.com/whitelisting.html) in [tailwind.config.js](./tailwind.config.js) or in the stylesheet itself by placing `/* purgecss ignore */` directly above the selector. Please use sparingly.
+If you find a class name is getting stripped out erroneously in production builds, it can be [whitelisted](https://purgecss.com/whitelisting.html) in [tailwind.config.js](./tailwind.config.js) or in the stylesheet itself by placing `/* purgecss ignore */` directly above the selector. Please use sparingly.
 
 #### Custom styles
 
 In the event you need to write styles that Tailwind is not able to affectively cover you should try to keep the following guidelines in mind:
 
-- SCSS is available.
-- PurgeCSS does not currently strip styles written outside of `src/styles/tailwind.css`, so it's up to you to keep styles limited to what you're using in your component.
-- Custom styles should be restricted to the component they are being used in. Typically, this is just an `index.scss` file adjacent to your component's `index.tsx` and then imported.
-- Because custom styles, once imported, are included in the global styles you should still be careful about how you're naming classes to avoid collisions. Generally you should match the class name to the name of your component. If you're using a shared component it's recommended to use `[component]-[descriptor]` style (e.g. using a shared UnitRow to display secondary emails could be `unit-row-secondary-emails`).
+- SCSS is available if it's absolutely necessary to add a custom style.
+- Custom styles should be restricted to the component they are being used in. Typically, this is just an imported `index.scss` file adjacent to your component's `index.tsx` and then imported.
```

Comment:
Yeah, further to my comment about directly importing PostCSS files into components, I can see this becoming a little confusing.